stories/LineChart.stories.js

```diff
@@ -735,6 +735,7 @@ storiesOf('LineChart', module)
             `${point.name}: ${Number.parseFloat(point.value).toFixed(3)}`,
           xLabel: point =>
             moment(point.timestamp).format('DD-MM-YYYY HH:mm:ss'),
+          position: Date.now() - 1000 * 10,
```

Comment:
Could you add this as a separate story? Or maybe just a second `<LineChart>` here. I'd like to be able to compare the behavior with and without this prop set.